go.mod

```diff
@@ -3,6 +3,15 @@ module github.com/DataWorkflowServices/dws-test-driver
 go 1.19
 
 require (
+	github.com/HewlettPackard/dws v0.0.1
```

Comment:
v0.0.6
It doesn't change the vendored content, but it should be changed anyway.